phpBB/phpbb/template/twig/loader.php

```diff
@@ -126,12 +126,12 @@ protected function findTemplate($name)
 
 		// First, find the template name. The override above of validateName
 		//	causes the validateName process to be skipped for this call
-		$file = parent::findTemplate($name);
+		$file = parent::findTemplate($name, $throw);
 
 		try
 		{
 			// Try validating the name (which may throw an exception)
-			parent::validateName($name);
+			$this->validateName($name);
```

Comment:
I didn't find a decent way to handle this, so that's temporary.
Method `validateName` is now private in `Twig_Loader_Filesystem` so the `parent::validateName()` can't be used anymore, but this breaks the current behavior (validateName was overwritten in this class so that call to `parent::findTemplate()` call this class implementation)
Any help here about how to handle this correctly would be greatly appreciated.

Reply:
As validateName does not use any state, you could just duplicate it in our loader... We could also try to rework the whole loader but it's a bit more complicated.

Reply:
@Nicofuma yes we can duplicate `validateName` in the loader, but when we call `parent::findTemplate`, it itself calls `validateName()` which is private (for twig 2.x) so this won't call the overriden implementation.